chia/util/db_wrapper.py

```diff
@@ -132,9 +159,12 @@ async def managed(
         log_path: Optional[Path] = None,
         journal_mode: str = "WAL",
         synchronous: Optional[str] = None,
-        foreign_keys: bool = False,
+        foreign_keys: Optional[bool] = None,
```

Comment:
it's not obvious what the distinction between `None` and `False` is. Why is this not just a `bool`?

Reply:
oh, `None` just means don't change anything. The DB may already have disabled foreign keys